twoliter/src/cmd/publish_kit.rs

```diff
@@ -41,11 +44,16 @@ impl PublishKit {
         install_tools(&toolsdir).await?;
         let makefile_path = toolsdir.join("Makefile.toml");
 
+        let publish_kit_repo = match &self.kit_repo {
```

Comment:
Should we not be populating this from the Twoliter.override file vs an argument?

Reply:
The Twoliter.override file knows about dependencies, but not about resultant kits.